autoPyTorch/api/base_task.py

```diff
@@ -542,8 +549,9 @@ def _do_traditional_prediction(self, time_for_traditional):
             memory_limit = int(math.ceil(memory_limit))
         available_classifiers = get_available_classifiers()
         dask_futures = list()
-        time_for_traditional_classifier_sec = int(time_for_traditional/len(available_classifiers))
-        for num_run, classifier in enumerate(available_classifiers, start=self._num_run+1):
+        time_for_traditional_classifier_sec = int(time_for_traditional / len(available_classifiers))
+        for num_run, classifier in enumerate(available_classifiers,
+                                             start=self._num_run + 1 if self._num_run is not None else 1):
```

Comment:
same here I think... probably it is more intuitive to just have num_run in argument. Return it to the main code and increase it as needed.

Reply:
okay I'll do this then